### PR TITLE
Update create-jira-issues-in-cd-stages.md

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ You can also [update Jira issues](/docs/continuous-delivery/x-platform-cd-featur
 
 ## Limitations
 
-While it's not a strict limitation, some users can forget that when you use a Jira Create step it creates a new, independent Jira issue every time it is run (as opposed to [updating](/docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages) the same issue).
-
-It is important to remember that you should only add Jira Create to a stage if you want to create a new Jira issue on every run of the stage.
+* Harness supports only Jira fields of type `Option`, `Array`, `Any`, `Number`, `Date`, and `String`.
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:19]: `User` needs to be added

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 2]
```diff
@@ -39,12 +41,11 @@ To add a Jira Create step, do the following:
 
 ## Add Issue fields
 
-In Jira fields, you can select specific fields within a Jira issue. For more information, go to [Jira custom fields](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/).
+In Jira fields, you can select specific fields within a Jira issue. For example, [multi value](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/) fields to allow users to select multiple options from a drop-down list.
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:44]: `For example, [multi value](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/) fields to allow users to select multiple options from a drop-down list.` can be added in limitation section: to have all field related points at a single place.

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md [hunk 3]
```diff
@@ -17,7 +17,10 @@ You can also [update Jira issues](/docs/continuous-delivery/x-platform-cd-featur
 
 ## Limitations
 
-* Harness supports only Jira fields of type `Option`, `Array`, `Any`, `Number`, `Date`, and `String`. Harness does not integrate with Jira fields that manage users, issue links, or attachments. This means that Jira fields like Assignee and Sprint are not accessible in Harness' Jira integration.
+* Harness supports only Jira fields of type `Option`, `Array`, `Any`, `Number`, `Date`, and `String`.
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md:20]: Add `User` here

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md [hunk 4]
```diff
@@ -17,7 +17,10 @@ You can also [update Jira issues](/docs/continuous-delivery/x-platform-cd-featur
 
 ## Limitations
 
-* Harness supports only Jira fields of type `Option`, `Array`, `Any`, `Number`, `Date`, and `String`. Harness does not integrate with Jira fields that manage users, issue links, or attachments. This means that Jira fields like Assignee and Sprint are not accessible in Harness' Jira integration.
+* Harness supports only Jira fields of type `Option`, `Array`, `Any`, `Number`, `Date`, and `String`. 
+* Harness does not integrate with Jira fields that manage issue links or attachments. 
+* You can add unsupported Jira fields using an HTTP step. For more details, go to [Use the HTTP step to set unsupported fields when creating Jira issues](/kb/continuous-delivery/articles/create-cascading-fields-jira/).
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md:22]: We can add that custom multi field line here, or cross-link the whole limitations for maintainability

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md [hunk 5]
```diff
@@ -17,7 +17,10 @@ You can also [update Jira issues](/docs/continuous-delivery/x-platform-cd-featur
...
+
 
 ## Add a Jira Update step
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md:25]: Here in add issue fields section, we can cross-link limitation of update (as done in create)

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md (hunk 2 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:44]:
```suggestion
In Jira fields, you can select specific fields within a Jira issue. 
```

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md (hunk 1 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:19]:
```suggestion
* Harness supports only Jira fields of type `User`, `Option`, `Array`, `Any`, `Number`, `Date`, and `String`. For example, [multi value](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/) fields to allow users to select multiple options from a drop-down list.
```

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md (hunk 2 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:44]:
```suggestion
In Jira fields, you can select specific fields within a Jira issue. For more information on custom fields, go to [Jira custom fields](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/).
```

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 6]
```diff
@@ -39,12 +41,11 @@ To add a Jira Create step, do the following:
...
 
-Harness supports only Jira fields of type `Option`, `Array`, `Any`, `Number`, `Date`, `User`, and `String`. 
-For example, Harness supports [multi value](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/) fields to allow users to select multiple options from a drop-down list.
+For more information on custom fileds, go to [Jira custom fields](https://support.atlassian.com/jira-cloud-administration/docs/custom-fields-types-in-company-managed-projects/).
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:46]:
```suggestion
```

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md (hunk 3 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-jira-issues-in-cd-stages.md:20]:
```suggestion
* Harness supports only Jira fields of type `User`, `Option`, `Array`, `Any`, `Number`, `Date`, and `String`. 
```

**Review by naman-goenka — APPROVED** [review]

**Pull request merged**